pkg/keystore/keystore.go

```diff
@@ -260,8 +247,8 @@ func NewTestKeystore() (*Keystore, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	systemDir := filepath.Join(dir, common.DefaultSystemConfigDir)
-	localDir := filepath.Join(dir, common.DefaultLocalConfigDir)
+	systemDir := filepath.Join(dir, "/usr/lib/rkt")
```

Comment:
I understand your rationale here but can you perhaps const these and add a comment that they should be kept in sync?